src/manifolds/ProbabilitySimplex.jl

```diff
@@ -281,6 +281,44 @@ mean(::ProbabilitySimplex, ::Any...)
 
 default_estimation_method(::ProbabilitySimplex, ::typeof(mean)) = GeodesicInterpolation()
 
+@doc raw"""
+    rand(::ProbabilitySimplex; vector_at=nothing, σ::Real=1.0)
+
+
+When `vector_at` is `nothing`, return a random (uniform over the Fisher-Rao metric)
+point `x` on the [`ProbabilitySimplex`](@ref) manifold `M` according to the isometric embedding into 
+the `n`-sphere by normalizing the vector length of a sample from a multivariate Gaussian. See [^Marsaglia1972].
+
+When `vector_at` is not `nothing`, return a (Gaussian) random vector from the tangent space
+``T_{p}\mathrm{\Delta}^n``by shifting a multivariate Gaussian with standard deviation `σ` 
+to have a zero component sum.
+
+[^Marsaglia1972]:
+    > Marsaglia, G.:
+    > _Choosing a Point from the Surface of a Sphere_.
+    > Annals of Mathematical Statistics, 43 (2): 645–646, 1972.
+    > doi: [10.1214/aoms/1177692644](https://doi.org/10.1214/aoms/1177692644)
+"""
+rand(::ProbabilitySimplex; σ::Real=1.0)
+
+function Random.rand!(
+    rng::AbstractRNG,
+    M::ProbabilitySimplex,
+    pX;
+    vector_at=nothing,
+    σ=one(eltype(pX)),
+)
+    if isnothing(vector_at)
```

Comment:
You transferred this to the rand with the Euclidean Metric which looks nice, but then this should be changed to the rand that Seth described at https://github.com/JuliaManifolds/Manifolds.jl/issues/606#issuecomment-1546869201 - both for point and vector?

Reply:
Otherwise we would double code, which is not nice – and the whole idea of moving this rand over is to have a second (i.e. different) one here :)

Reply:
I think they are different `rand` for point right? one samples uniformly from l1 ball and the other from l2 ball. happy to change the vector as well of course, but not sure I fully understand how the tangent space geometry changes with the two different metrics

Reply:
I think it would be easiest to do `change_metric!(M, pX, EuclideanMetric(), vector_at, pX)` to convert to Fisher-Rao metric.